grazer/src/main/webapp/WEB-INF/views/fileUploadForm.jsp

```diff
@@ -0,0 +1,53 @@
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<div id="global">
+    <form:form modelAttribute="transformation" commandName="transformation" action="/grazer/transformation/save?${_csrf.parameterName}=${_csrf.token}" method="post" enctype="multipart/form-data">
```

Comment:
you need either modelAttribute or commandName, not both (apparently modelAttribute is preferred)

Reply:
also, you can't hard code the action. our production environment app is called ep-grazer for example, so this would break.